docs/tsa-policy.md

```diff
@@ -356,27 +351,6 @@ NOTE: It is advisable that, when obtaining a time-stamp token, the subscriber
 verifies that the time-stamp token has been correctly signed and that the
 private key used to sign the time-stamp token has not been compromised.
 
-## 6.3. Relying Party Obligations
```

Comment:
Should we leave this section in but remove a reference to 7.1.2? It seems like this is still useful to include guidance for a relying pary

Reply:
Yeah, makes sense.